### ISSUE TITLE
System Tray icons are far apart

### ISSUE THREAD

**Issue by szmarczak** [solus-project/budgie-desktop#1932]
## Budgie version

budgie-desktop 10.5.1

## Operating system

```
Linux solus 5.4.12-144.current #1 SMP PREEMPT Mon Jan 20 15:46:35 UTC 2020 x86_64 GNU/Linux
```

## Steps to reproduce the issue

1. Open `Budgie Desktop Settings`.
2. Enter the `Bottom panel` tab.
3. Move the `System Tray` applet up and down.

## Actual result

There is some sort of left padding I think. Just look how the tray icons are far apart:

![image](https://user-images.githubusercontent.com/36894700/73190433-49be6a80-4126-11ea-9f9b-be84c1f41740.png)

## Expected result

The tray icons should be located next to each other.

**Comment by Umlatt** [comment]
If this isn't on a fresh Solus install, you can try remove the system tray applet from the panel completely, reboot, then re-add it.

I had all sorts of weird behavior with the system tray applet (started several months ago).
Since I did the above "hacky workaround", the system tray panel has been well behaved (been running well for about a week (on my daily driver).

Not really a "fix" to the problem, but (unless yours is a fresh install), I think the issue is actually due to a code change in a past upgrade, rather than an actual existing code bug.

**Comment by szmarczak** [comment]
> If this isn't on a fresh Solus install

No, it's not.

> I think the issue is actually due to a code change in a past upgrade, rather than an actual existing code bug.

But I'm running the latest Solus OS, so it is a code bug.

**Comment by Umlatt** [comment]
Sorry for the misunderstanding.
What I was trying to say was that the bug is in a past system-tray upgrade process, rather than the current system tray code (based on what my experience has been so far, since I applied the "work around").

Have you tested the "work around" at all?

**Comment by szmarczak** [comment]
> Have you tested the "work around" at all?

Yep. Nothing changes.

**Comment by Umlatt** [comment]
It's a pity that it did not work for you.
Thanks for the response.

So it looks like that workaround is only for icon overlap.

**Comment by JoshStrobl** [comment]
Should be resolved as of #1998 :tada:

Thanks @serebit
